configurations/test_pipeline_configuration.py

```diff
@@ -142,7 +142,14 @@
                     ),
                     CodingConfiguration(
                         code_scheme=load_code_scheme("age_category"),
-                        analysis_dataset="age_category"
+                        analysis_dataset="age_category",
+                        age_categories = {
+                            (10, 14): "10 to 14",
+                            (15, 17): "15 to 17",
+                            (18, 35): "18 to 35",
+                            (36, 54): "36 to 54",
+                            (55, 99): "55 to 99"
+                        }
```

Comment:
I think this is ok and happy to keep like this so we can keep making progress.

The slight hesitations I have are that
i. this means we need to specify the categories every time, rather than relying on an avf default.
ii. the functional programmer in me instinctively leans to something like code_imputations=[ImputeAgeCategories(categories=...), ...] as it makes adding additional imputations easier, but that might well be overkill here

Reply:
on i)  We've seen this vary between projects in Cesc depending on the target audience. Moving this here standardizes _impute_age_category() so we update the categories & scheme between projects.
ii) My view is storing imputation configs in their resp coding_plan makes it easier to trace computations that happen in each analysis dataset.